src/components/CtaCard/CtaCardItem.js

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+
+const CtaCardItem = ({ label }) => {
+  return (
+    <div className="CtaCardItem">
+      {label}
+    </div>
+  );
+};
+
+CtaCardItem.propTypes = {
+  label: PropTypes.node,
```

Comment:
Are You sure that this is node (html element) ? Maybe it's string. If label could be node write something like this
` label: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.node,
  ]),`

Reply:
A string is a node mate :D

Reply:
Got it